File: Basic Coding Problems/01 Fundamentals/017 Write a program to enter Principal, Time and Rate of Interest and calculate Simple Interest/C/simple_interest.c
#include <stdio.h>

int main() {
    float principal, time, rate_of_interest, simple_interest;

    printf("Enter the principal amount: ");
    scanf("%f", &principal);

    printf("Enter the time period in years: ");
    scanf("%f", &time);

    printf("Enter the interest rate: ");
    scanf("%f", &rate_of_interest);

    simple_interest = principal * time * rate_of_interest / 100;

    printf("The simple interest is: %f\n", simple_interest);

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/017 Write a program to enter Principal, Time and Rate of Interest and calculate Simple Interest/Java/simple_interest.java
import java.util.Scanner;

public class simple_interest {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        // Get the principal amount, time, and interest rate from the user.
        System.out.print("Enter the principal amount: ");
        float principal = input.nextFloat();

        System.out.print("Enter the time period in years: ");
        float time = input.nextFloat();

        System.out.print("Enter the interest rate: ");
        float rateOfInterest = input.nextFloat();

        // Calculate the simple interest.
        float simpleInterest = principal * time * rateOfInterest / 100;

        // Print the simple interest.
        System.out.println("The simple interest is: " + simpleInterest);
    }
}